### PR TITLE
[Logs FAQ] How to remap custom severity values

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
This article explains how custom status level can be remapped to the official log status

### Motivation
The category remapper was recently added. This was the missing piece to allow such a feature.

### Preview link
https://docs-staging.datadoghq.com/nils/log-faq-custom-status-remapper/

### CODE REVIEW

**Comment by NBParis** [comment]
I know I should do it but if you could put the image in the repo that would be awesome.

File: content/logs/faq/how-to-remap-custom-severity-values-to-the-official-log-status.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- link: "logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+---
+
+By default, the [Log Status Remapper][1] relies on the [Syslog severity standards][2].  
+However there might be other systems having different severity values that you might want to remap on the official log status.
+This is possible thanks to the [Category Processor][3] that defines a mapping between your custom values and the expected ones.
+
+In this article develop how to do this with 2 examples: Bunyan levels and web access logs.
```
Comment (estib) [inline content/logs/faq/how-to-remap-custom-severity-values-to-the-official-log-status.md]: Sounds a bit funny. Better:
> In this article, we show how

File: content/logs/faq/how-to-remap-custom-severity-values-to-the-official-log-status.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+{{< img src="logs/faq/category_processor.png" alt="Category processor" responsive="true" popup="true">}}
+
+Then add a status remapper that uses the newly created attribute:
+
+{{< img src="logs/faq/log_status_remapper.png" alt="log status remapper" responsive="true" popup="true">}}
+
+## Bunyan levels
+
+Bunyan levels are similar to the Syslog one but with a value multiplied by 10.
```
Comment (estib) [inline content/logs/faq/how-to-remap-custom-severity-values-to-the-official-log-status.md]: Better:
> Bunyan levels are similar to those of Syslog, but their values are multiplied by 10.

Comment (NBParis) [reply content/logs/faq/how-to-remap-custom-severity-values-to-the-official-log-status.md]: Done, thanks

File: content/logs/faq/how-to-remap-custom-severity-values-to-the-official-log-status.md (hunk 1 above)
Comment (NBParis) [reply content/logs/faq/how-to-remap-custom-severity-values-to-the-official-log-status.md]: Done, thanks

**Pull request merged**